app/page.tsx

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) ZeroC, Inc.
```

Comment:
It's fine not to include it, since it's the home page,
but this page doesn't have the feedback buttons on it anymore, when it used to.
Again, just letting you know in case this wasn't intentional.
![image](https://github.com/icerpc/icerpc-docs/assets/25963603/fa968635-f9d7-4777-aacc-c4883607bb95)

Reply:
This was also intentional. Only was on the homepage because we were hijacking the docs rendering.